lib/tasks/restore/access_token.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+namespace :restore do
```

Comment:
Are we planning on having/moving other tasks under `lib/tasks/restore`?

Reply:
Well I could just create without namespace but I have no idea

Reply:
it could be `access_tokens:master:update` wdyt?